### DISCUSSION TITLE
Pagination not working on specific pages

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [livewire/livewire#1111] [4. General]
Here is the html of the pagination that I currently have. Every time I move away from **Page 2**, the click event for **Page 2** on the pagination doesn't work. The click event of all the other pages seems to work fine except for **Page 2**. 

I seem to have the same issue with the second last page on the pagination, so if I have 80 pages, Page 79  isn't clickable after navigating away from that page.

To resolve this, I have to go back to **Page 1** and refresh the page and then **Page 2** works. Again I navigate away from **Page 2**, it stops working, Could someone advice if I am missing something?

https://gist.github.com/abishekrsrikaanth/0e879c26a6240b495502f547b9eec149

I have also added the blade file to the gist that generates this pagination. 
The problem is that I am also not able to debug it because the event never gets triggered when clicking on the page.

I do see this error at times on the console and am not sure it's related

```
directive.js:48 Uncaught TypeError: Cannot read property 'match' of null
    at _default.parseOutMethodAndParams (directive.js:48)
    at _default.get (directive.js:20)
    at node_initializer.js:149
    at Component.value (index.js:421)
    at HTMLButtonElement.<anonymous> (node_initializer.js:141)
```

Appreciate it if someone could help figure out what the issue may be. It's been driving me a bit crazy.

**Comment by abishekrsrikaanth** [comment]
I was able to solve this by adding id's to the elements on the pagination. There is a thread on the issue list mentioning the same and I tried it and works fine. https://github.com/livewire/livewire/issues/872#issuecomment-644583616

**Reply by lorisleiva** [reply]
Thanks! That fixed my issue too. 🙇‍♂️
